Danggn/Screen/Home/Controller/ListItemViewController.swift

```diff
@@ -8,10 +8,105 @@
 import UIKit
 
 class ListItemViewController: UIViewController {
-
+    
+    @IBOutlet weak var listCollectionView: UICollectionView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        setNavigationUI()
+        registerCell()
+    }
+    
+    @IBAction func itemCreateButtondidTap(_ sender: Any) {
```

Comment:
Button이 호출을 해주는 메소드이니 UIButton으로 명시화를 해주는건 어떨까요?
```suggestion
    @IBAction func itemCreateButtondidTap(_ sender: UIButton) {
```

Reply:
이부분 수정하겠습니다 감사합니다!

Reply:
이제야 확인한건데 didTap에서 DidTap이라고 적었어야되는데  실수로 didTap이라고 썼는데 lint에서는 감지가 안된건가요?

Reply:
SwiftLint에서는 아쉽게도 특정 함수 네이밍 까지는 파싱해주지는 않습니다
우리 프로젝트에 대한 lint rule은 같이 회의에서 정한 것 처럼 
https://github.com/THE-SOPT-Joint-Seminar-Danggn/Daangn-iOS#swiftlint-rule 에 명시되어있습니다:)

Reply:
대소문자와 관련한 rule은 https://realm.github.io/SwiftLint/identifier_name.html 해당 룰이 그나마 가장 근접한 것 같네요:)